Cmake travis failure in master

https://travis-ci.com/github/CorsixTH/CorsixTH/builds/188955436

```
/home/travis/build/CorsixTH/CorsixTH/CorsixTH/CppTest/test_main.cpp:2:10: fatal error: catch2/catch.hpp: No such file or directory

 #include "catch2/catch.hpp"

          ^~~~~~~~~~~~~~~~~~

compilation terminated.

CorsixTH/CppTest/CMakeFiles/UnitTests.dir/build.make:62: recipe for target 'CorsixTH/CppTest/CMakeFiles/UnitTests.dir/test_main.cpp.o' failed

make[2]: *** [CorsixTH/CppTest/CMakeFiles/UnitTests.dir/test_main.cpp.o] Error 1

make[2]: Leaving directory '/home/travis/build/CorsixTH/CorsixTH/fresh'

CMakeFiles/Makefile2:499: recipe for target 'CorsixTH/CppTest/CMakeFiles/UnitTests.dir/all' failed

make[1]: *** [CorsixTH/CppTest/CMakeFiles/UnitTests.dir/all] Error 2

make[1]: Leaving directory '/home/travis/build/CorsixTH/CorsixTH/fresh'

Makefile:129: recipe for target 'all' failed

make: *** [all] Error 2

The command "make VERBOSE=1" exited with 2.
```

```
1: Test command: 

Unable to find executable: /home/travis/build/CorsixTH/CorsixTH/fresh/CorsixTH/CppTest/UnitTests

1/1 Test #1: UnitTests ........................***Not Run   0.00 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.00 sec

The following tests FAILED:

	  1 - UnitTests (Not Run)

Errors while running CTest

The command "eval 'ctest -VV'" exited with 8.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -24,7 +24,7 @@ addons:
 before_install:
   # Bionic does not have catch2 in its repo so clone and install locally
   - cd $HOME
-  - git clone https://github.com/catchorg/Catch2.git --depth=1
+  - git clone https://github.com/catchorg/Catch2.git -b v2.13.2 --depth=1
   - cd Catch2
   - cmake -Bbuild -H. -DBUILD_TESTING=OFF
   - cd build && sudo make install

